packages/astro/src/core/util.ts

```diff
@@ -72,6 +73,13 @@ export function resolveDependency(dep: string, astroConfig: AstroConfig) {
   return pathToFileURL(resolved).toString();
 }
 
-export function viteifyPath(pathname: string): string {
-  return `/@fs/${pathname.replace(/^\//, '')}`;
+/**
+ * Vite-ify URL
+ * Given a file URL, return an ID that matches Vite’s module graph. Needed for resolution and stack trace fixing.
+ * Must match the following format:
+ * Linux/Mac:  /Users/astro/code/my-project/src/pages/index.astro
+ * Windows:    C:/Users/astro/code/my-project/src/pages/index.astro
+ */
+export function viteifyURL(filePath: URL): string {
+  return slash(fileURLToPath(filePath));
```

Comment:
Change to the `viteifyURL` function: before, we were prefixing everything with `/@fs`. I found removing this, and providing this format to have a better success rate in matching with Vite’s module graph.

If this causes any problems let me know. But I couldn’t find any instance of us requiring `/@fs/*` in Astro.